src/store/state.ts

```diff
@@ -187,7 +187,7 @@ export function preparePipeline(pipeline: Pipeline, state: VQBState) {
   }
   const { interpolateFunc, variables, pipelines } = state;
   if (pipelines && Object.keys(pipelines).length) {
-    pipeline = dereferencePipelines(pipeline, pipelines);
+    pipeline = dereferencePipelines(pipeline, pipelines) as Pipeline;
```

Comment:
@davinov , in any case we will have to force a type, here pipeline is reaffected so it causes issue (before Pipeline after DereferencedPipeline :cry: ). Remove all Old Pipeline occurencies will take a lot of time and i'm not sure were i'm able to keep original string Pipeline type

Reply:
@alice-sevin Oh I understand, this may drag this ticket into a bigger refactoring than we expected.
So maybe the best option is to merge this as is, with the necessary `as` and discriminating conditions between pipeline reference and pipeline dereferenced, and to add an issue about making Pipeline into two different types.
We can do that later on.